### DISCUSSION TITLE
Could someone post instructions to getting this running on Linux ?

### DISCUSSION THREAD

**Discussion by WilsonBradley** [xbapps/xbvr#469] [General]
Could someone post instructions to getting this running on Linux ?
Raspberry Pi would be ideal..

Using rclone (instead of Raidrive in Windows)
Have Gdrive mounted but not the correct shared folder.

**Comment by snickers2k** [comment]
either docker or
download the tar.gz for your system, unzip tar -xvzf *.tar.gz 
move it to your system "mv xbvr /usr/local/bin/xbvr"
run it with "xbvr"
